Source/core/Proxy.h

```diff
@@ -1032,7 +1029,7 @@ namespace Core {
                     ProxyObjectType* baseElement(const_cast<ProxyObjectType*>(this));
 
                     baseElement->__Relinquish<ELEMENT>();
```

Comment:
Think this line should be changed as well but please check all usages of all HAS_MEMBER methods in this file as mentioned above